pkg/summarizer/summary.go

```diff
@@ -858,15 +858,16 @@ func failingTestSummaries(rows []*statepb.Row, template *configpb.LinkTemplate,
 			FailureMessage:    alert.FailureMessage,
 			PassBuildId:       alert.PassBuildId,
 			// TODO(fejta): better build info
-			BuildLink:          alert.BuildLink,
-			BuildLinkText:      alert.BuildLinkText,
-			BuildUrlText:       alert.BuildUrlText,
-			LinkedBugs:         row.Issues,
-			FailTestLink:       buildFailLink(alert.FailTestId, row.Id),
-			LatestFailTestLink: buildFailLink(alert.LatestFailTestId, row.Id),
-			Properties:         alert.Properties,
-			HotlistIds:         alert.HotlistIds,
-			EmailAddresses:     alert.EmailAddresses,
+			BuildLink:           alert.BuildLink,
+			BuildLinkText:       alert.BuildLinkText,
+			BuildUrlText:        alert.BuildUrlText,
+			LinkedBugs:          row.Issues,
+			FailTestLink:        buildFailLink(alert.FailTestId, row.Id),
+			LatestFailTestLink:  buildFailLink(alert.LatestFailTestId, row.Id),
+			Properties:          alert.Properties,
+			CustomColumnHeaders: alert.CustomColumnHeaders,
```

Comment:
Note that 'alert' has to pull info from somewhere first; this is set in the Updater in alertRow() (https://github.com/GoogleCloudPlatform/testgrid/blob/main/pkg/updater/updater.go#L1281).